tests/openvino/utils_tests.py

```diff
@@ -145,7 +146,7 @@
     "qwen2": "fxmarty/tiny-dummy-qwen2",
     "qwen2_moe": "katuni4ka/tiny-random-qwen1.5-moe",
     "qwen2_vl": "katuni4ka/tiny-random-qwen2vl",
-    "qwen2_5_vl": "katuni4ka/tiny-random-qwen2.5-vl",
+    "qwen2_5_vl": "optimum-internal-testing/tiny-random-qwen2.5-vl",
```

Comment:
https://huggingface.co/katuni4ka/tiny-random-qwen2.5-vl is not compatible with transformers >= v4.55 coming from
https://github.com/huggingface/transformers/blob/v4.55.0/src/transformers/models/qwen2_5_vl/modeling_qwen2_5_vl.py#L1219

optimum-internal-testing/tiny-random-qwen2.5-vl has the same parameter with `out_hidden_size` set to 16 to keep compatibility with transformers >= v4.55